src/dotnet/Chat.UI.Blazor/Components/ChatPlaybackToggle.razor

```diff
@@ -25,13 +27,32 @@
         var isPlayingActive = await ChatUI.IsPlaying.Use(cancellationToken).ConfigureAwait(false);
         var chatPlaybackState = await ChatPlayers.ChatPlaybackState.Use(cancellationToken).ConfigureAwait(false);
         var isPlayingHistorical = chatPlaybackState is HistoricalChatPlaybackState;
+        if (!isPlayingActive)
+            return new Model() {
+                IsPlayingActive = isPlayingActive,
+                IsPlayingHistorical = isPlayingHistorical,
+            };
+        var listenedChatIds = await ChatUI.ListenedChatIds.Use(cancellationToken).ConfigureAwait(false);
+        ChatUI.ListenedChatIds.Value = AddToListened(listenedChatIds);
```

Comment:
Logic to manage listened chats list should not be here. May be better to put it in ChatUI service.
Consult with me or @alexyakunin

Reply:
Yep, note the right place, but also you shouldn't normally modify the state from Compute Methods.